arm-machinelearning/2016-05-01-preview/swagger/webservices.json

```diff
@@ -0,0 +1,915 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure ML Web Services Management Client",
+    "description": "These APIs allow end users to operate on Azure Machine Learning Web Services resources. They support the following operations:<ul><li>Create or update a web service</li><li>Get a web service</li><li>Patch a web service</li><li>Delete a web service</li><li>Get All Web Services in a Resource Group </li><li>Get All Web Services in a Subscription</li><li>Get Web Services Keys</li></ul>",
+    "version": "2016-05-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json",
+    "text/json"
+  ],
+  "produces": [
+    "application/json",
+    "text/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearning/webServices/{webServiceName}": {
+      "put": {
+        "tags": [
+          "WebServices"
+        ],
+        "operationId": "WebServices_CreateOrUpdateWebService",
+        "description": "Create a new Azure ML web service or update an existing one.",
+        "parameters": [
+          {
```

Comment:
subscriptionId, resourceGroupName and webServiceName parameters are not defined in the operation